<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordDialog</class>
 <widget class="QDialog" name="PasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Continous function</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Function:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="functionLineEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_mathMLPreview">
       <property name="text">
        <string>MathML</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Dimensions:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="dimensionSpinBox">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="lblPassword">
       <property name="text">
        <string>Variables( csv ) </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="variableLineEdit">
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="colorPickerWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Range:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QDoubleSpinBox" name="RangeXSpinBox"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="RangeYSpinBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PasswordDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PasswordDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
